src/libs/CMakeLists.txt

```diff
@@ -138,19 +142,19 @@ endif(CUPS_FOUND)
 
 # Add libs references
 foreach(module ${MODULES})
-	target_link_libraries(${module} lib_darktable)
+  target_link_libraries(${module} lib_darktable)
 endforeach(module)
 
 # Add installation target for each module
 # TODO: Modules are now not only lighttable related and should be placed
 #       in a better matching structure then just in lighttable directory.
 foreach(module ${MODULES})
-    if (WIN32)
-      _detach_debuginfo (${module} ${CMAKE_INSTALL_LIBDIR}/darktable/plugins/lighttable)
-    else()
-        set_target_properties(${module}
-                              PROPERTIES
-                              INSTALL_RPATH ${RPATH_ORIGIN}/../..)
-    endif(WIN32)
-    install(TARGETS ${module} DESTINATION ${CMAKE_INSTALL_LIBDIR}/darktable/plugins/lighttable COMPONENT DTApplication)
+  if(WIN32)
+    _detach_debuginfo (${module} ${CMAKE_INSTALL_LIBDIR}/darktable/plugins/lighttable)
+  else()
+    set_target_properties(${module}
+                          PROPERTIES
+                          INSTALL_RPATH ${RPATH_ORIGIN}/../..)
+  endif()
```

Comment:
BTW, now seeing that... It would have been better to keep the `endif(WIN32)` for clarity, no?

Reply:
Yes please, it is easier to find the matching `endif()` to me, so better keep this style. TIA.